README.md

```diff
@@ -113,10 +113,10 @@ Our promise is that we'll follow what's common practice in the Kubernetes world
 
 The OpenTelemetry Operator *might* work on versions outside of the given range, but when opening new issues, please make sure to test your scenario on a supported version.
 
-| OpenTelemetry Operator | Kubernetes           |
+| Splunk Otel Operator   | Kubernetes           |
```

Comment:
hypernit, `OTel` or `OpenTelemetry`

Reply:
Understandable. I made some updates with your input in mind.